Applications/Utils/FileConverter/generateMatPropsFromMatID.cpp

```diff
@@ -69,7 +69,7 @@ int main (int argc, char* argv[])
     std::string const name =
         BaseLib::extractBaseNameWithoutExtension(mesh_arg.getValue());
     // create file
-    std::string const new_matname(name + "prop_");
```

Comment:
these must not be changed. This is not a single occurrence. You will have to update your scripts.

Reply:
It is gone with my fixing of this afternoon. I think you forgot to fetch and update.

Reply:
Sorry. My bad.